src/message-media.ts

```diff
@@ -140,6 +147,9 @@ export class MediaMessage extends Message {
   }
 
   public filename(): string {
```

Comment:
According to <https://nodejs.org/api/path.html#path_path_parse_path>, I think we should rename `filename()` at here to `fileBase()`, return a full file base name(without path) like `music.mp3`